webapp/client/src/index.js

```diff
@@ -43,6 +44,7 @@ const allowedComponents = {
   TelephoneNumberPage,
   StmindSelectionPage,
   ConfirmationPage,
+  LogoutPage,
```

Comment:
Personally, I would put that page between line 30 and 31 to have a more logical order of the pages (logout after login).

Reply:
i agree, will be done!